docs/site/Appsody-LoopBack.md

```diff
@@ -0,0 +1,990 @@
+---
+lang: en
+title: 'Deploying to Cloud with Appsody'
```

Comment:
The title should be `Developing and Deploying LoopBack Applications with Appsody`.